### PR TITLE
feat: OPTIC-1242: Ask new OSS installs how they found about us

### PR DESCRIPTION
**Pull request by yyassi-heartex**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Backend (Database)
- [x] Backend (API)
- [x] Frontend

### Describe the reason for change
adding an optional field on signup to ask how the new user heard about Label Studio
updated signup will look like this:
image

### CODE REVIEW

**Comment by yyassi-heartex** [comment]
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/11893524444)
> Successfully merged:  8 files changed, 26 insertions(+), 72 deletions(-)

File: label_studio/users/forms.py [hunk 1]
```diff
@@ -17,6 +17,16 @@
...
+FOUND_US_ELABORATE = 'Other'
+FOUND_US_OPTIONS = (
+    ('Gi', 'Github'),
+    ('Em', 'Email or newsletter'),
+    ('Se', 'Search engine'),
+    ('Fr', 'Friend or coworker'),
+    ('Ad', 'Ad'),
+    ('Ot', FOUND_US_ELABORATE),
+)
+
```
Comment (bmartel) [inline label_studio/users/forms.py:29]:
```suggestion
```

Are these used anymore?

File: label_studio/users/forms.py [hunk 2]
```diff
@@ -85,8 +97,16 @@ def save(self):
...
+        how_find_us = None
+        elaborate = None
         if 'allow_newsletters' in cleaned:
             allow_newsletters = cleaned['allow_newsletters']
+        if 'how_find_us' in cleaned:
+            how_find_us = cleaned['how_find_us']
+        if 'elaborate' in cleaned and how_find_us == FOUND_US_ELABORATE:
+            elaborate = cleaned['elaborate']
+
+        print('----> UserSignupForm save cleaned', cleaned, how_find_us, elaborate)
```
Comment (bmartel) [inline label_studio/users/forms.py]:
```suggestion
```

**Review by bmartel — APPROVED** [review]
just a print statement to remove, rest looks good!

**Review by hlomzik — APPROVED** [review]

**Pull request merged**